index.js

```diff
@@ -94,18 +95,44 @@ function fastifyStatic (fastify, opts, next) {
   // Set the schema hide property if defined in opts or true by default
   const schema = { schema: { hide: typeof opts.schemaHide !== 'undefined' ? opts.schemaHide : true } }
 
-  if (opts.serve !== false) {
-    fastify.get(prefix + '*', schema, function (req, reply) {
-      pumpSendToReply(req, reply, '/' + req.params['*'])
-    })
-  }
-
   if (opts.decorateReply !== false) {
     fastify.decorateReply('sendFile', function (filePath) {
       pumpSendToReply(this.request, this, filePath)
     })
   }
 
+  if (opts.serve !== false) {
+    if (opts.wildcard === undefined || opts.wildcard === true) {
+      fastify.get(prefix + '*', schema, function (req, reply) {
+        pumpSendToReply(req, reply, '/' + req.params['*'])
+      })
+    } else {
+      glob(path.join(sendOptions.root, '**/*'), function (err, files) {
```

Comment:
we can use the [sync method](https://www.npmjs.com/package/glob#globsyncpattern-options).

Reply:
why? fastify boots asynchronously, so it should be ok to do this.

Reply:
It is simplest.

Reply:
It encourages a wrong pattern of booting synchronously.